OneLauncher/Settings.py

```diff
@@ -275,8 +279,12 @@ def SaveSettings(self, saveAccountDetails=None, savePassword=None, game=None):
             tempNode.appendChild(doc.createTextNode("False"))
         gameConfigNode.appendChild(tempNode)
 
+        tempNode = doc.createElementNS(EMPTY_NAMESPACE, "Client")
+        tempNode.appendChild(doc.createTextNode("%s" % (self.client)))
+        gameConfigNode.appendChild(tempNode)
+
         tempNode = doc.createElementNS(EMPTY_NAMESPACE, "x64Client")
-        if self.x64ClientEnabled:
+        if self.client == "WIN64":
             tempNode.appendChild(doc.createTextNode("True"))
         else:
```

Comment:
This should only be set to False for the WIN32 client and just not appended otherwise.